ElectronicObserver/Data/TsunDbSubmission/TsunDbEntity.cs

```diff
@@ -2,23 +2,31 @@
 using System.IO;
 using System.Net.Http;
 using System.Net.Http.Headers;
+using System.Text.Json;
+using System.Text.Json.Serialization;
 using System.Threading.Tasks;
 using ElectronicObserver.Utility;
-using Newtonsoft.Json;
 
-namespace ElectronicObserver.Data;
+namespace ElectronicObserver.Data.TsunDbSubmission;
 
 public abstract class TsunDbEntity
 {
-	[JsonIgnore]
 	/// <summary>
 	/// Is initialized ? 
 	/// </summary>
+	[JsonIgnore]
 	public bool IsInitialized { get; set; } = true;
 
 	protected abstract string Url { get; }
 
-	protected virtual bool IsBetaAPI { get; } = false;
+	protected virtual bool IsBetaAPI => false;
+
+
+	private static readonly JsonSerializerOptions JsonSerializer = new()
```

Comment:
Field naming would be _jsonSerializer. You could also just turn this into a property.

Reply:
turned into a static property